lib/vagrant-ovirt3/config.rb

```diff
@@ -17,6 +17,10 @@ class Config < Vagrant.plugin('2', :config)
       attr_accessor :console
       attr_accessor :disk_size
 
+      attr_accessor :ca_no_verify
+      attr_accessor :ca_cert_store
```

Comment:
Can you add a comment above this:
# TODO: change to 'ca_cert' once rbovirt PR #55 merges.